stubs/livewire/resources/views/api/api-token-manager.blade.php

```diff
@@ -94,7 +94,7 @@
     @endif
 
     <!-- Token Value Modal -->
-    <x-jet-dialog-modal show="displayingToken">
+    <x-jet-dialog-modal wire:model="displayingToken">
```

Comment:
To me `wire:model`, even if it's not actually used as `wire:model` in the underlying component is the best way to express data binding to inputs.

> Note: this also supports `wire:model.defer`